app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -27,6 +32,23 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
 
+    fun fetchEventSpeakers(id: Long) {
+        speakerService.fetchSpeakersForEvent(id)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe ({
+                //Do Nothing
+            }, {
+                Timber.e(it, "Error fetching speaker for event id %d", id)
+                mutableError.value = resource.getString(R.string.error_fetching_event_message)
+            })
+    }
+
+    fun loadEventSpeakers(id: Long): LiveData<List<Speaker>> {
+        val speakerList = speakerService.fetchSpeakersFromDb(id)
```

Comment:
Store this in the ViewModel. What use is this variable, you're just returning it